docs/node/daemon/config/settings.mdx

```diff
@@ -317,6 +369,35 @@ max_tx_bytes = 4194304
 
 # Size of the cache (used to filter transactions we saw earlier) in transactions
 cache_size = 10000
+
+#######################################################
+###         State Sync Configuration Options        ###
+#######################################################
+[chain.statesync]
+# State sync rapidly bootstraps a new node by discovering, fetching, and restoring the database
+# snapshot from peers instead of fetching and replaying historical blocks. Requires some peers in
+# the network to take and serve snapshots. State sync is not attempted if the node
+# has any local state (LastBlockHeight > 0). The node will have a truncated block history,
+# starting from the height of the snapshot.
+enable = true
+
+# Path to the directory where the received snapshot is stored
```

Comment:
If relative, it will be under the root directory.

Reply:
its relative to the current working directory where kwild is run